tasks/build/download_node_builds.js

```diff
@@ -50,11 +119,9 @@ module.exports = function (grunt) {
       } else {
         await fromNode(cb => {
           resp
-          .pipe(createGunzip())
-          .on('error', cb)
-          .pipe(new Extract({ path: downloadDir, strip: 1 }))
+          .pipe(createWriteStream(resolve(downloadDir, file)))
           .on('error', cb)
-          .on('end', cb);
+          .on('finish', cb);
```

Comment:
Couldn't this if/else be collapsed into a single expression now? I think `file` should === `node.exe` on windows, and if that's the cause I think the logic in the if and else are identical?